### ISSUE TITLE
How to reproduce the performance of f1-score=80.83 on ICDAR2015?

### ISSUE THREAD

**Issue by zxDeepDiver** [argman/EAST#35]
Hi,

I can't reproduce the 80.83 f1score when directly run `python multigpu_train.py --gpu_list=0 --input_size=512 --batch_size_per_gpu=14 --checkpoint_path=/tmp/east_icdar2015_resnet_v1_50_rbox/ \
--text_scale=512 --training_data_path=/data/ocr/icdar2015/ --geometry=RBOX --learning_rate=0.0001 --num_readers=24 \
--pretrained_model_path=/tmp/resnet_v1_50.ckpt` on the ICDAR2015+2013 training images.

Could you please tell me the parameter configurations of your experiment that achieves 80.83 f1-score?
about:  1. the batch size per GPU; 2. the number of GPUs ;3. the initial learning rate 4. the number of steps that you train your model for.

Thank you very much.

**Comment by argman** [comment]
whats your performance ? I think the default configuration can reproduce that, in about 10000 steps 14*4gpu.

**Comment by zxDeepDiver** [comment]
Thank you.

**Comment by sdsy888** [comment]
Hi, @argman , I found that the annotation data(gt_*.txt) icdar13 is different from icdar15.(icdar13 gt files only has 4 coodinates, and icdar 15 has 8 coods)
And your code in `icdar.py` seems like only support reading ground_truth file of icdar15 format. So can you tell me how do you train the EAST on both icdar15 and icdar13?

**Comment by viig99** [comment]
```python
tlx, tly, brx, bry = list(map(float, line[:4]))
x1, y1, x2, y2, x3, y3, x4, y4 = tlx, tly, brx, tly, brx, bry, tlx, bry
```

**Comment by argman** [comment]
@tianzhuotao , thats cool, can you share your model or ideas ?

**Comment by duanjiaqi** [comment]
@argman , Hi, Could you please tell me the MS result on ICDAR2015 test set?

**Comment by wangyuxin87** [comment]
> by improving the model a little bit, the f1-score can reach nearly 82 on ICDAR2015 test set (trained with ICDAR2013 AND ICDAR2015 training data)

can you show your idea? Thanks a lot! My email: 741764216@qq.com

**Comment by lsabrinax** [comment]
> by improving the model a little bit, the f1-score can reach nearly 82 on ICDAR2015 test set (trained with ICDAR2013 AND ICDAR2015 training data)

can you share your idea?Thank you!
